app/models/publisher.rb

```diff
@@ -71,6 +72,7 @@ class Publisher < ApplicationRecord
 
   scope :email_verified, -> { where.not(email: nil) }
   scope :not_admin, -> { where.not(role: ADMIN) }
+  scope :not_partner, -> { where.not(role: PARTNER) }
```

Comment:
how do you see this scope being used? thinking that we should add `publisher` `partner` and `admin` scopes and tests for the scopes.

Reply:
Thanks for the review @nvonpentz , you're cool with calling it `partner` right? 

For uphold, one idea might be to prevent them from reaching the dashboard, but I'm not sure if this completely prevents them, as they might be able to connect through APIs. Lmk if you have any other ideas

I added that scope because it seemed like a good idea we might want to use later, but good thinking I'll add some tests for these scopes.

Reply:
We could chain `Publisher.not_admin.not_partner` or use `Publisher.publisher`